docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+* Documenten die bij een zaak horen kunnen dan in de DRC worden opgehaald mbv het zaaknummer. Idem voor besluit(en) per zaak en betrokkenen.
+* Dit komt sterk overeen met het concept Linked-data
+Opmerking AK Wat is de overeenkomst dan? M.i. is het simpelweg een objecten- of informatiemodel met objecten en relaties. LinkedData is dan een manier om gebruik te kunnen maken van de in die modellen onderscheiden relaties. De Zakenregistratiecomponent is dus niet meer één grote database met daarin alle gegevens over een zaak. Maar meer een kernregistratie van zaken met veel links (URL’s) naar objecten in andere registraties (dan wel verwijzen andere objecten naar zaken).
+Opmerking AK Essentie hiervan is dat objecten die nu in het RGBZ en daarmee in de ZRC voorkomen maar ook relevantie hebben in een andere context of domein, als aparte registratie neergezet worden om recht te doen aan de betekenis als zelfstandige objecttype.
+* Dit kunnen zijn (niet uitputtend):
+    * Kernregistratie Klanten en Contacten (voor initiators,  belanghebbenden, gemachtigden e.d.)
+    * Kernregistratie medewerkers (o.a. voor behandelaars)
+    * Kernregistratie besluiten (besluiten)
+* Documentregistratiecomponent (documenten cq. informatieobjecten)
+    * Zaaktyperegistratiecomponent (Zaaktype, Statustypen, Resultaattypen,..)
+    * Domein/objectregistraties (domein/objectgegevens; meerdere registraties: één per domein/proces)
+    * Meldingenregistratie (ontvangen aanvragen en meldingen)
+    * Basisregistraties (personen, bedrijven, panden, terreinen, percelen e.d.)
+    * Evt. andere kernregistraties (Zaakobjecten die nog niet genoemd zijn, maar wel behoren tot kerngegevens (meervoudig gebruikt))
+
+
+
+Figuur 2: ZGW in een gegevenslandschap
+
+## 1.4 Vraagstukken
+
+Bovenstaande plaat schetst het concept ZGW in een gegevenslandschap al heel aardig.
+
+Toch zijn er nog enkele openstaande vragen te beantwoorden. Voor een deel is dit al af te leiden uit de stippellijnen in bovenstaande plaat:
+
+### 1.4.1. Hoe verhoudt melding/verzoek zich tot een zaak?
+
+In bovenstaande figuur is de relatie tussen Melding/Aanvraag (Verzoek) en Zaak als een stippellijn getekend. Dit omdat Verzoek niet in het RGBZ voorkomt (zat wel in GFO Zaken) en we dus niet weten hoe Verzoek en Zaak zich tot elkaar verhouden. Essentie van de relatie lijkt te zijn dat een verzoek of melding een op zich zelf staand ‘iets’ is en behandeld wordt als zaak. Één verzoek kan leiden tot meerdere zaken. Meerdere Verzoeken kunnen ook leiden tot één Zaak?
```

Comment:
Deze zgn. business objects zijn tot nu toe niet gemodelleerd en gestandaardiseerd. In eerdere discussies hebben we gezegd dat dat wel de bedoeling is. Dat is veel werk. En leveranciers zullen zelf met modellen komen. Je zou kunnen vragen dat die objecten niet in de ZAC's worden opgeslagen maar in separate componenten en dat hiervoor API's worden beschikbaar opgesteld en gepubliceerd. Zodat deze gegevens iig kunnen worden benaderd vanuit andere componenten.